test/integrations/destinations/linkedin_ads/router/data.ts

```diff
@@ -0,0 +1,519 @@
+                    gender: 'non-binary',
+                    db: '19950715',
+                    lastname: 'Rudderlabs',
+                    firstName: 'Test',
+                    address: {
+                      city: 'Kolkata',
+                      state: 'WB',
+                      zip: '700114',
+                      country: 'IN',
+                    },
+                  },
+                  device: {
+                    advertisingId: 'abc123',
+                  },
+                  library: {
+                    name: 'rudder-sdk-ruby-sync',
+                    version: '1.0.6',
+                  },
+                },
+                messageId: '7208bbb6-2c4e-45bb-bf5b-ad426f3593e9',
+                timestamp: '2024-02-10T12:16:07.251Z',
+                properties: {
+                  tax: 2,
+                  total: 27.5,
+                  coupon: 'hasbros',
+                  revenue: 48,
+                  price: 25,
+                  quantity: 2,
+                  currency: 'USD',
+                  discount: 2.5,
+                  order_id: '50314b8e9bcf000000000000',
+                  requestIP: '123.0.0.0',
+                  optOutType: 'LDP',
+                  clickId: 'dummy_clickId',
+
+                  shipping: 3,
+                  subtotal: 22.5,
+                  affiliation: 'Google Store',
+                  checkout_id: 'fksdjfsdjfisjf9sdfjsd9f',
+                },
+                anonymousId: '50be5c78-6c3f-4b60-be84-97805a316fb1',
+                integrations: {
+                  All: true,
+                },
+              },
+              metadata: {
+                sourceType: '',
+                destinationType: '',
+                namespace: '',
+                jobId: 3,
+                secret: {
+                  accessToken: 'dummyToken',
+                },
+              },
+              destination: commonDestination,
+            },
+          ],
+          destType: 'linkedin_ads',
+        },
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: [
+            {
+              metadata: [
+                {
+                  sourceType: '',
+                  destinationType: '',
+                  namespace: '',
+                  jobId: 3,
+                  secret: {
+                    accessToken: 'dummyToken',
+                  },
+                },
+              ],
+              destination: {
+                ID: '12335',
+                Name: 'sample-destination',
+                DestinationDefinition: {
+                  ID: '123',
+                  Name: 'linkedin_ads',
+                  DisplayName: 'LinkedIn Ads',
+                  Config: {
+                    cdkV2Enabled: true,
+                  },
+                },
+                WorkspaceID: '123',
+                Transformations: [],
+                Config: {
```

Comment:
Can we try to split config following latest strategy that we follow

Reply:
done, LMK if you meant this